tests/unit/parsers/test.manifestjson.js

```diff
@@ -2401,19 +2401,36 @@ describe('ManifestJSONParser', () => {
           return { code, dataPath, file, message, description };
         });
 
-        const commonErrorProps = { ...messages.MANIFEST_THEME_LWT_ALIAS };
+        const commonErrorProps = {
```

Comment:
These doesn't seem to be `commonErrorProps` anymore, as they are then overwritten a few lines below for every one of the items in the `expectedErrors` array.

At a first glance it seems that the previous definition of `commonErrorProps` (` { ...messages.MANIFEST_THEME_LWT_ALIAS };`) should still work.

am I reading it wrong?

Reply:
Actually, I removed that whole part and simply hardcoded the expected error messages. That makes it much easier to reason about.